code/src/components/ToggleQuestions.js

```diff
@@ -0,0 +1,191 @@
+    setUserInput("");
+    setPopulationAge("");
+    setYearsToMars("");
+    setFutureChoice("");
+    setQuestionNumber("");
+    setMoreFutureChoices([]);
+    setIsSubmit(false);
+  };
+
+  return (
+    <>
+      {/*If the isSubmit !== true (eg it's false) */}
+      {!isSubmit && (
+          <main>
+          {/*While isSubmit is false the question with the corresponding number
+        of questionNumber will be shown*/}
+          {questionNumber === "Question1" && (
+            <section className="question" aria-labelledby="Question1">
+              <h2 className="question-heading" id="Question1">
+                Your thoughts
+              </h2>
+              <Question1 userInput={userInput} setUserInput={setUserInput} />
+            </section>
+          )}
+
+          {questionNumber === "Question2" && (
+            <section className="question" aria-labelledby="Question2">
+              <h2 className="question-heading" id="Question2">
+                We keep getting older...
+              </h2>
+              <Question2
+                min="0"
+                max="100"
+                popluationAge={populationAge}
+                setPopulationAge={setPopulationAge}
+              />
+            </section>
+          )}
+
+          {questionNumber === "Question3" && (
+            <section className="question" aria-labelledby="Question3">
+              <h2 className="question-heading" id="Question3">
+                What about space?
+              </h2>
+              <Question3
+                valueArray={[
+                  "0-10 years",
+                  "10 to 20 years",
+                  "over 20 years",
+                  "over 100 years",
+                  "Never",
+                ]}
+                yearsToMars={yearsToMars}
+                setYearsToMars={setYearsToMars}
+              />
+            </section>
+          )}
+
+          {questionNumber === "Question4" && (
+            <section className="question" aria-labelledby="Question4">
+              <h2 className="question-heading" id="Question4">
+                If the Sci-Fi movies were right...
+              </h2>
+              <Question4
+                optionArray={[
+                  "synthetic food",
+                  "flying cars",
+                  "cyborg society",
+                  "cool gadgets",
+                ]}
+                futureChoice={futureChoice}
+                setFutureChoice={setFutureChoice}
+              />
+            </section>
+          )}
+
+          {questionNumber === "Question5" && (
+            <section className="question" aria-labelledby="Question5">
+              <h2 className="question-heading" id="Question5">
+                Interesting, tell me more!
+              </h2>
+              <ToggleSubQuestions
+                futureChoice={futureChoice}
+                setMoreFutureChoices={setMoreFutureChoices}
+                moreFutureChoices={moreFutureChoices}
```

Comment:
It seems like I could skip choosing a checkbox here on question 5 and could still click on Submit.